src/org/myworldgis/netlogo/ProjectLatLon.java

```diff
@@ -0,0 +1,127 @@
+package org.myworldgis.netlogo;
+
+import org.nlogo.api.Argument;
+import org.nlogo.api.Context;
+import org.nlogo.api.ExtensionException;
+import org.nlogo.api.LogoException;
+import org.nlogo.api.LogoListBuilder;
+import org.nlogo.core.Syntax;
+import org.nlogo.core.SyntaxJ;
+import org.myworldgis.projection.Ellipsoid;
+import org.myworldgis.projection.Geographic;
+import org.myworldgis.projection.Projection;
```

Comment:
I'd prefer to keep the imports clustered by their package name, so these `org.myworldgis` entries can move up with the `org.myworldgis.netlogo` at the top.